tst/schedulers/transfer_learning/test_rush.py

```diff
@@ -0,0 +1,197 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+import numpy as np
+import pandas as pd
+import pytest
+
+from syne_tune.backend.trial_status import Trial
+from syne_tune.optimizer.schedulers.hyperband_rush import RUSHStoppingRungSystem
+from syne_tune.optimizer.schedulers.transfer_learning import TransferLearningTaskEvaluations
+from syne_tune.optimizer.schedulers.transfer_learning.rush import RUSHScheduler
+from syne_tune.search_space import randint
+
+
+@pytest.fixture()
+def config_space():
+    return {
+        'steps': 10,
+        'm': randint(0, 20),
+        'b': randint(-100, 100)
+    }
+
+
+@pytest.fixture()
+def scheduler(metadata, config_space, request):
+    return RUSHScheduler(config_space=config_space,
+                         metric='loss',
+                         max_t=10,
+                         type=request.param,
```

Comment:
Yes, I've added this param as well as tests for 'promotion' already. I wanted to avoid rewriting all tests later given that I already know that these tests will come. But those tests don't test for promotion yet.